Tests/test_Mafft_tool.py

```diff
@@ -87,7 +87,7 @@ def test_Mafft_simple(self):
         # became "Progressive alignment 1/2..." and "Progressive alignment 2/2..."
         self.assertTrue(("Progressive alignment ..." in stderrdata) or
                         ("Progressive alignment 1/" in stderrdata), stderrdata)
-        self.assertTrue("$#=0" not in stderrdata)
+        self.assertNotIn("$#=0" , stderrdata)
```

Comment:
Just noticed it, I'm on it. 

There was another fail, [https://travis-ci.org/biopython/biopython/jobs/182623109](https://travis-ci.org/biopython/biopython/jobs/182623109)

I've changed the line to the original version and added a TODO since I was unsure of the cause of the error.